packages/instantsearch-ui-components/src/components/TrendingFacets.tsx

```diff
@@ -0,0 +1,162 @@
+/** @jsx createElement */
+
+import { cx } from '../lib';
+
+import {
+  createDefaultEmptyComponent,
+  createDefaultHeaderComponent,
+  createDefaultItemComponent,
+} from './recommend-shared';
+
+import type {
+  ComponentProps,
+  RecommendClassNames,
+  RecommendInnerComponentProps,
+  RecommendItemComponentProps,
+  RecommendStatus,
+  RecommendTranslations,
+  RecordWithObjectID,
+  Renderer,
+  SendEventForHits,
+} from '../types';
+import type { TrendingFacetHit } from 'algoliasearch';
```

Comment:
this should also be in the helper or just retyped `{ objectID: string, facetValue: string}`?